packages/feedback-service/src/resolver.ts

```diff
@@ -16,7 +17,26 @@ export const FeedbackResolver = {
     },
     getFeedbackBy(root: any,  args : any, ctx: any){
       return Feedback.find(args.input).exec();
-    }
+    },
+    getAllJiraIssues(root: any,  args : any, ctx: any) {
+    const jira = new JiraApi({
+    protocol: 'https',
+    host: process.env.JIRA_HOST,
+    username: process.env.JIRA_USERNAME,
+    password: process.env.JIRA_PASSWORD,
+    apiVersion: '2',
+    strictSSL: false
+   });
+   return  jira.getIssuesForBoard("3844")
+      .then(function(getIssuesForBoard: any){
+      return getIssuesForBoard['issues'].map( (issue: any) => {
+          issue['fields']['ticketID'] = issue['key'];
+          return issue['fields']
+          }) 
+      })
+      .catch((err:any) => err);          
```

Comment:
A catch should either handle the error in a sensible way, or throw it. Never return it, as that may cause any further `catch` to be ignored, and it gets ppassed to the next `.then` instead.
```suggestion
      .catch(err => {
          throw err;
      });          
```